query-engine/connector-test-kit-rs/query-engine-tests/tests/new/interactive_tx.rs

```diff
@@ -426,7 +426,8 @@ mod itx_isolation {
     use query_engine_tests::*;
 
     // All (SQL) connectors support serializable.
-    #[connector_test(exclude(MongoDb))]
+    // TiDB support snapshot isolation, but didn't support serializable isolation.
+    #[connector_test(exclude(MongoDb, TiDB))]
```

Comment:
Link for this?

Reply:
Link: https://docs.pingcap.com/tidb/stable/transaction-isolation-levels#tidb-transaction-isolation-levels

> TiDB implements Snapshot Isolation (SI) consistency, which it advertises as REPEATABLE-READ for compatibility with MySQL. T

Can I add this link to the comment in the code?

Reply:
Yes, I think that would be helpful to be able to understand👍

Reply:
https://github.com/prisma/prisma-engines/pull/3308/commits/3ef6694c8918133efc0b2e59393709e538fcf48d